src/delegation_backend/app_config.go

```diff
@@ -106,6 +120,16 @@ func getEnvChecked(variable string, log logging.EventLogger) string {
 	return value
 }
 
+func boolEnvChecked(variable string) bool {
+	value := os.Getenv(variable)
+	switch {
+	case value == "1":
+		return true
+	default:
+		return false
+	}
+}
```

Comment:
I suggest to make it even more strict:
```suggestion
	switch (value) {
	case "1":
		return true
        case "0":
		return false
	default:
	        log.Fatalf("%s, if set, should be either 0 or 1!", variable)
	}
}
```
This will need having `log` as an additional argument to this function.

Reply:
Ok, fair enough!